content/reference-architectures/cold-start.md

```diff
@@ -0,0 +1,627 @@
+make init
+
+# Build docker image
+make docker/build
+
+# Install the wrapper shell
+make install
+
+# Run the shell
+root.cloudposse.co
+
+# Login to AWS as the admin user with your MFA device
+assume-role
+```
+{{% /dialog %}}
+
+You should see the `Docker` image built, `geodesic` shell started, and after you run the `assume-role` command, you will be logged in to the `root` account as the `admin` user:
+
+```
+# Starting new root.cloudposse.co session from cloudposse/root.cloudposse.co:latest
+# Exposing port 36682
+* Started EC2 metadata service at http://169.254.169.254/latest
+                     _         _                 _
+     _ __ ___   ___ | |_   ___| | ___  _   _  __| |_ __   ___  ___ ___  ___   ___ ___
+    | '__/ _ \ / _ \| __| / __| |/ _ \| | | |/ _` | '_ \ / _ \/ __/ __|/ _ \ / __/ _ \
+    | | | (_) | (_) | |_ | (__| | (_) | |_| | (_| | |_) | (_) \__ \__ \  __/| (_| (_) |
+    |_|  \___/ \___/ \__(_)___|_|\___/ \__,_|\__,_| .__/ \___/|___/___/\___(_)___\___/
+                                                  |_|
+
+IMPORTANT:
+* Your $HOME directory has been mounted to `/localhost`
+* Use `aws-vault` to manage your sessions
+* Run `assume-role` to start a session
+
+Add your local private SSH key to the key chain. Hit ^C to skip.
+Enter passphrase for /localhost/.ssh/id_rsa:
+Identity added: /localhost/.ssh/id_rsa (/localhost/.ssh/id_rsa)
+-> Run 'assume-role' to login to AWS
+ ⧉  root.cloudposse.co
+❌   (none) ~ ➤  assume-role
+Enter passphrase to unlock /conf/.awsvault/keys/:
+Enter token for arn:aws:iam::323330167063:mfa/admin@cloudposse.co: 172432
+* Assumed role arn:aws:iam::323330167063:user/admin@cloudposse.co
+* Found SSH agent config
+ ⧉  root.cloudposse.co
+✅   (cpco-root-admin) ~ ➤
+```
+
+## Provision `tfstate-backend` Project For `root`
+
+We store Terraform state in an S3 bucket and use a DynamoDB table for state locking (allowing many users to work on the same project without affecting each other and corrupting the state).
+
+But we don't have the state bucket and DynamoDB table provisioned yet, so we can't store Terraform state in them.
+
+We will create the bucket and table using local state, and then import the state into the bucket.
+
+Execute this sequence of steps in the `root` geodesic session:
+
+{{% dialog type="code-block" icon="fa fa-code" title="Provision `tfstate-backend` Project For `root`" %}}
+```
+cd tfstate-backend
+
+Comment out the `backend          "s3"             {}` section in `tfstate-backend/main.tf`
+
+Run `init-terraform`
+
+Run `terraform plan` and then `terraform apply`
+
+Re-enable `backend          "s3"             {}` section in `tfstate-backend/main.tf`
+
+Re-run `init-terraform`
+
+Re-run `terraform apply`, answer `yes` when asked to import state
+
+```
+{{% /dialog %}}
+
+{{% dialog type="info" icon="fa-info-circle" title="Note" %}}
+You could use the following commands to comment out and then uncomment the `backend` section:
+
+```shell
```

Comment:
Update to use igor's new script.

https://github.com/cloudposse/terraform-root-modules/pull/15

Reply:
will do it in the next PR after the scripts PR gets reviewed, tested and approved.
otherwise, too many changes in this PR